### ISSUE TITLE
issue: Maybe wrong directory

### ISSUE THREAD

**Issue by mutalisk999** [matter-labs/zksync-era#332]
https://github.com/matter-labs/zksync-era/blame/677611e5ad813fb93d9bdebaa8d82d738e8bb971/core/bin/system-constants-generator/src/main.rs#L213

```rust
fn update_rust_system_constants(intrinsic_gas_constants: &IntrinsicSystemGasConstants) {
    let rust_fee_constants = generate_rust_fee_constants(intrinsic_gas_constants);
    save_file(
        "core/lib/config/src/constants/fees/intrinsic.rs",
        rust_fee_constants,
    );
}
```

Maybe it should be "core/lib/constants/src/fees/intrinsic.rs"

**Comment by popzxc** [comment]
Thanks, you're probably right.

**Comment by mutalisk999** [comment]
It is my pleasure. LOL
Has been fixed in PR #336.
